grunt/watch.js

```diff
@@ -1,60 +1,36 @@
+"use strict";
+
 module.exports = {
-    options:
-    {
-        atBegin: true,
-        livereload: true,
+    options: {
         event: [ "changed", "added", "deleted" ]
     },
-    angular:
-    {
-        files: "<%= concat.angular.src %>",
-        tasks: [ "concat:angular" ]
+    index: {
+        files: "<%= copy.index.src %>",
+        tasks: [ "copy:index" ]
     },
-    components:
-    {
-        files: "<%= concat.components.src %>",
-        tasks: [ "concat:components" ]
+    js: {
+        files: "<%= jshint.dev.src %>",
+        tasks: [ "newer:jshint:dev", "browserify:build" ]
     },
-    scripts:
-    {
-        files: "<%= concat.scripts.src %>",
-        tasks: [ "envLocal", "newer:jshint:dev", "concat:scripts", "replace" ]
-    },
-    styles:
-    {
-        files: "source/modules/*/styles/*.*",
-        tasks: [ "less", "replace", "postcss" ]
-    },
-    index:
-    {
-        files: "source/modules/_app/templates/index.html",
-        tasks: [ "envLocal", "includeSource", "ejs", "replace" ]
-    },
-    templates:
-    {
+    templates: {
         options: { cwd: "<%= ngtemplates.build.cwd %>" },
         files: "<%= ngtemplates.build.src %>",
-        tasks: [ "envLocal", "ngtemplates", "replace" ]
+        // TODO: cut browserify out of this?
```

Comment:
I got that much, but I'm certain it'd be possible to reference the template file in `index.html` then when deploying, join it with the browserified files. I figured it'd get complicated and just left this TODO here